common/fs/src/cache/watch.rs

```diff
@@ -2,7 +2,15 @@ use inotify::{EventMask, Inotify, WatchDescriptor, WatchMask};
 use std::ffi::OsString;
 use std::io;
 use std::path::Path;
+use std::sync::Arc;
 
+use futures::future::Either;
+use futures::{Stream, StreamExt};
+use tokio::time::Instant;
+
+use tokio::sync::Mutex;
+
+const INOTIFY_EVENT_GRACE_PERIOD_MS: u64 = 1000;
```

Comment:
I know we're probably gonna play around with this value, but 1 second seems a bit excessive of a starting point. I imagine a move event split across multiple events will appear a couple milliseconds from each other.

Reply:
I've reduced it down to 100ms, that still feels really big, but I havn't been able to find documentation anywhere on what the maximum could be.

The only current art I could find is [notify-rs](https://github.com/notify-rs/notify/blob/80204c6f246b08278065b8cb5362630b10423fcf/src/inotify.rs#L384), which uses 10ms. Should I drop it further down to match that? I'm not entirely sure what the repercussions are either way